### PR TITLE
feat: ExportFeature

### PR DESCRIPTION
**Pull request by vinnyjth**

### CODE REVIEW

File: packages/apollos-ui-connected/src/ShareButtonConnected/ShareButtonConnected.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ const ShareButtonConnected = ({
     {({ track }) => (
       <Query query={GET_SHARE_CONTENT} variables={{ nodeId: nodeId || itemId }}>
         {({ data, loading }) => {
+          console.log(data, loading);
```
Comment (vinnyjth) [inline packages/apollos-ui-connected/src/ShareButtonConnected/ShareButtonConnected.js:25]:
```suggestion
```

**Comment by vinnyjth** [comment]
@conrad-vanl could I get some UI/UX feedback please?

**Comment by vinnyjth** [comment]
Thanks :) 

All 1.0 Components - used a stylesheet for positioning but otherwise it's 100% untouched components from our old ui-kit.

File: packages/apollos-ui-connected/src/features/ExportFeature.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, StyleSheet } from 'react-native';
+
+import { ActionCard, BodyText } from '@apollosproject/ui-kit';
+import ShareButtonConnected from '../ShareButtonConnected';
+
+const styles = StyleSheet.create({
```
Comment (redreceipt) [inline packages/apollos-ui-connected/src/features/ExportFeature.js:8]: 😍

Comment (vinnyjth) [reply packages/apollos-ui-connected/src/features/ExportFeature.js:8]: I knew I was going either get loved or eviscerated for this 😂

**Review by redreceipt — APPROVED** [review]

File: packages/apollos-ui-connected/src/features/ExportFeature.js (hunk 2 above)
Comment (redreceipt) [reply packages/apollos-ui-connected/src/features/ExportFeature.js:8]: you knew what I would say

Comment (vinnyjth) [reply packages/apollos-ui-connected/src/features/ExportFeature.js:8]: merge before @calebpanza sees it

Comment (calebpanza) [reply packages/apollos-ui-connected/src/features/ExportFeature.js:8]: _**adds to the list of components to rewrite in 2.0_

Comment (calebpanza) [reply packages/apollos-ui-connected/src/features/ExportFeature.js:8]: 😂😂😂 jkjk!

This looks really cool! Great job on this!

**Comment by calebpanza** [comment]
Only note is that it looks like the shadow might be getting clipped out of frame?

**Comment by camrun91** [comment]
Looks good to me

**Pull request merged**